### ISSUE TITLE
Cannot build release, fails with error[E0432]: unresolved import `clap::Clap`

### ISSUE THREAD

**Issue by CristianCantoro** [dariost/CodeColosseum#1]
Hi,

## Description

Trying to build the code with:
```
cargo build --release
```
The build fails with the following error:
```
   Compiling CodeColosseum v0.0.1 (.../CodeColosseum)
error[E0432]: unresolved import `clap::Clap`
 --> src/client.rs:4:5
  |
4 | use clap::Clap;
  |     ^^^^^^^^^^ no `Clap` in the root

...

error: could not compile `CodeColosseum` due to 8 previous errors
```

Full error log: [issue.log](https://github.com/dariost/CodeColosseum/files/9526160/issue.log)

## Environment

* Operating System: GNU/Linux Ubuntu 18.04.6 LTS
* Rust version: 1.63.0 (4b91a6ea7 2022-08-08)

**Comment by CristianCantoro** [comment]
Trying to build the programs with Docker, I get the same error:

```
 --> src/client.rs:4:5
  |
4 | use clap::Clap;
  |     ^^^^^^^^^^ no `Clap` in the root

error: cannot determine resolution for the derive macro `Clap`

...

error: could not compile `CodeColosseum` due to 8 previous errors
```

Here's the Dockerfile:
```Dockerfile
FROM rust:1.63.0

RUN git clone https://github.com/dariost/CodeColosseum.git /opt/CodeColosseum

WORKDIR /opt/CodeColosseum

RUN cargo build --release

CMD ["/bin/bash"]
```

I build it with the following command:
```
docker build -t cristiancantoro/codecolosseum .
```
Full logs:
* [stdout.log](https://github.com/dariost/CodeColosseum/files/9526505/stdout.log)
* [stderr.log](https://github.com/dariost/CodeColosseum/files/9526506/stderr.log)

**Comment by dariost** [comment]
Fixed with commit [ce5bb81](https://github.com/dariost/CodeColosseum/commit/ce5bb81c572829934fb455924b1431961e2282c8)
